app/services/experiments/copy_experiment_as_template_service.rb

```diff
@@ -0,0 +1,103 @@
+# frozen_string_literal: true
+
+module Experiments
+  class CopyExperimentAsTemplateService
```

Comment:
I do believe this whole class can be DRYed up together with the `MoveToProjectService`, because many things are repeating. May I suggest creating a parent class where common logic would reside?

Reply:
If we decide for full refactoring (`move_to_proj`, `clone_proj`, `generate_workflow_image` + `Service.rb` module) with upper ideas, maybe we should open a new JIRA ticket and have separate PR for that?  
@jbargu 
@Ducz0r

Reply:
Hmm, I see 2 things here (which I agree with you @urbanrotnik that we should probably open a separate task/s about it):
* extension of existing `Service` with some additional interfacing (e.g. `success`, `errors`, ...) that could be usable for any service
* introducing a parent Service just for `CopyExperimentAsTemplateService`, `MoveExperimentService`, just to DRY these 2 services a bit.